images/armhf/python2-unicornhat.dev/README-FR.md

```diff
@@ -1,12 +1,12 @@
 Image Docker de développement pour le Unicorn sur Raspberry PI

```

Comment:
Removed and saved as a Gist : https://gist.github.com/actuino/5a5e93c2f9832cb3f9b31f446e1345db